src/routes/admins.js

```diff
@@ -1,3 +1,100 @@
-export default(apiServer) => {
-    
+import { list, readOne, updateOne, deleteOne } from 'mongoose-crudl'
+import jwt from 'jsonwebtoken';
+import AdminModel from '../models/Admin.js'
+import MethodNotAllowedError from '../errors/MethodNotAllowedError.js';
+import allowAccessTo from '../helpers/allowAccessTo.js'
+import crypto from 'crypto';
+
+export default (apiServer) => {
+  const secrets =  process.env.SECRETS.split(' ');
+
+  apiServer.get('/v1/admins/', async req => {
+    try {
+          allowAccessTo(req, secrets, [{ type: 'admin' }]);
+          const response = await list(AdminModel)
+          response.result.map(user => {
+            if (!user.password) { // if user didn't have password that means he still didn't accept the invitation
+              user.invitedtionStatus = "didn't accept yet";
```

Comment:
Great, but I would simply use a boolean value, for example invitationAccepted, or something like that

Reply:
you will also need to assign the result of the .map to response.result.items (and yes, you will need to call the .map on the .items